src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -107,6 +107,9 @@ public static bool IsDrawingSupported
 
         public static bool IsNotIntMaxValueArrayIndexSupported => s_largeArrayIsNotSupported.Value;
 
+        public static bool IsAssemblyLoadFromSupported => !IsNativeAot;
```

Comment:
Should we call it `IsDynamicAssemblyLoadSupported` or `IsAssemblyLoadingSupported` ? I assume that it will also apply to `Load(Stream)`, `LoadFile(string)`, etc.